src/main/java/seedu/duke/command/PurgeCommand.java

```diff
@@ -0,0 +1,15 @@
+package seedu.duke.command;
+
+import seedu.duke.company.CompanyList;
+
+public class PurgeCommand extends Command{
```

Comment:
perhaps a space can be added between `Command` and `{` for the codestyle to be consistent throughout